ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -775,28 +780,20 @@ func ConvertToVolumes(volumeConfigs map[string]*config.VolumeConfig) *volumes {
 	if volumeConfigs != nil {
 		for name, config := range volumeConfigs {
 			if config != nil {
+				// NOTE: If Driver field is not empty, this
+				// will add a prefix to the named volume on the container
 				if config.Driver != "" {
-					log.WithFields(log.Fields{
-						"volume name": name,
-						"option name": "driver",
-					}).Warn("Skipping unsupported YAML option...")
+					return nil, errors.New("Volume driver is not supported")
 				}
+				// Driver Options must relate to a specific volume driver
 				if len(config.DriverOpts) != 0 {
-					log.WithFields(log.Fields{
-						"volume name": name,
-						"option name": "driver_opts",
-					}).Warn("Skipping unsupported YAML option...")
-				}
-				if config.External.External {
```

Comment:
do we no longer need the check for `External` ?

Reply:
If the other two fields are not present but the config is still not nil, the only thing left is the `External` subfield. I was running into issues testing the value of this field explicitly (see [previous commment](https://github.com/aws/amazon-ecs-cli/pull/506#discussion_r187752598), and it seemed clunky and probably even more confusing to say `if config.External.External || !config.External.External`.